App/Sagas/NodeStarted.ts

```diff
@@ -2,14 +2,14 @@ import { call, put, take } from 'redux-saga/effects'
 import { getType } from 'typesafe-actions'
 import AccountActions from '../Redux/AccountRedux'
 import ContactsActions from '../Redux/ContactsRedux'
-import PreferencesActions from '../Redux/PreferencesRedux'
 import TextileNodeActions from '../Redux/TextileNodeRedux'
 import PhotoViewingActions from '../Redux/PhotoViewingRedux'
 import { getSDKVersion } from './NodeLifecycle'
 
 export function * onNodeStarted () {
-  while (yield take([getType(TextileNodeActions.startNodeSuccess), getType(PreferencesActions.onboardedSuccess)])) {
+  while (yield take([getType(TextileNodeActions.startNodeSuccess), getType(AccountActions.initSuccess)])) {
```

Comment:
Don't want to rely on the ui's onboarding success, here just rely on if the account was initialized yet or not. I think the sdk is going to want a different 'stop' / 'start' state than the one returned by the node, that simply lets the app turn off the node long term even though it's init'd. will cross that bridge later.